domain-management/src/main/java/org/jboss/as/domain/management/security/SecurityRealmAddHandler.java

```diff
@@ -160,6 +160,11 @@ protected void installServices(final OperationContext context, final String real
             addPlugInLoaderService(realmName, plugIns, serviceTarget);
         }
         InjectedSetValue<CallbackHandlerService> injectorSet = securityRealmService.getCallbackHandlerService();
+
+        // install the managed server realm callback handle for domain server authkeys
+        DomainManagedServerCallbackHandler dmc = new DomainManagedServerCallbackHandler();
+        dmc.installServerSecurityRealm(context, securityRealmService, serviceTarget, realmBuilder, injectorSet.injector());
```

Comment:
Ok, cool thanks. I've made additional changes, and cleaned it up quite a bit. I have a couple of things left to debug, and I'll reopen.